RFC-676-Contributor-Council-Charter.md

```diff
@@ -0,0 +1,221 @@
+request.
+  * It is preferred that community-led features are code-reviewed by a
+[Distinguished Contributor](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#badges-pilot-program) before AWS reviews them.
+    * If a Distinguished Contributor is not available within 30 days, AWS will review.
+    * The feature needs to adhere to all AWS development and security processes, which AWS will publish on the CDK GitHub. AWS is committed to
+providing all the required resources for timely reviews and merging.
+    * Those who build a feature through to completion are recognized in a manner to be determined.
+
+## Structure of the Council
+
+The Council is led by two Chairpersons. They do not submit opinions in the decision-making process:
+
+* One Co-Chair and the facilitator of Council sessions will come from the AWS Open Source Strategy & Marketing (OSSM) team.
+* The other Co-Chair is the AWS CDK Product Lead. They set the priorities for AWS resources, but do not have veto power over what the community can
+or can’t work on.
+
+Amazon will have five permanent members on the Council, which AWS may rotate as necessary. They will represent various AWS teams with vested
+interests in CDK to bring a multiplicity of viewpoints and experiences to Council debate (Engineering, Support, OSSM).
+
+The community will have five elected members on the Council.
+
+* The community representatives will be nominated and chosen by the CDK community through a public, time-constrained mechanism which permits
+transparency into the mechanism and allows the community to self-identify—and make an informed decision about—people who are actively looking to
+make deeper investment into CDK.
+* In order to participate in the full-breadth of conversation related to CDK and the project’s roadmap, Council members will be required to sign an
+NDA.
+* The five elected community representatives serve one-year terms. There is no limit to the number of subsequent terms a Council member may serve.
+Depending on community input, the AWS CDK team may decide to allocate a certain number of Council seats to specific community groups, such as the
+Open Construct Foundation, or Enterprise Customers. The rest of the seats would be undesignated.
+* We expect Council members to regularly attend Council meetings. If a Council member has two unannounced absences in a row, they will be removed
+and the Council will decide how to replace them.
+
+## First priority of the Council
+
+After the Council is elected, our desire is for the Council to immediately get to work evolving CDK. Their first tasks may include but are not
+limited to:
+
+* Co-definition of the Council (new processes, working groups, etc.)
+* Contributing to the AWS CDK roadmap
+* Building a prioritized list of proposed changes for debate
+
+# FAQ
+
+## What are we launching today?
+
+Today, AWS announces the formation of a Contributor Council, which will serve as a trusted advisor to AWS on CDK’s vision, direction, and
```

Comment:
your not launching anything today.  This is not a product release. Your asking for feedback on an idea.

Reply:
That wording comes from the AWS way of writing these docs. The PR/FAQ

Reply:
It does. Its totally inapproporate is'nt it. THis is not a PR.